### ISSUE TITLE
ValueError

### ISSUE THREAD

**Issue by geek-APTX4869** [zwyang6/SeCo#8]
`segs = F.interpolate(segs, size=labels.shape[1:], mode='bilinear', align_corners=False)`报错Input and output must have the same number of spatial dimensions, but got input with spatial dimensions of [28, 28] and output size of torch.Size([281, 500, 3]). Please provide input tensor in (N, C, d1, d2, ...,dK) format and output size in (o1, o2, ...,oK) format

**Comment by zwyang6** [comment]
Thanks for your attention. please check the size of your labels since the size 281, 500, 3 looks like from the image, not the label.
